### PR TITLE
Feature/example html css

### PR DESCRIPTION
**Pull request by lanhuong4867**
Do example from book

### CODE REVIEW

File: html-css/examples/chapter10.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <!-- USING EXTERNAL CSS -->
+    <link rel="stylesheet" href="css/chapter10.css">
+    <!-- USING INTERNAL CSS -->
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/chapter10.html]: Pls not write uppercase if it is not very important content.

`<!-- Using external CSS-->` is good enough

File: html-css/examples/css/chapter16.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+     #66cccc);
+    /* Opera 11.10+ */
+    background-image: -o-linear-gradient(#336666,
+     #66cccc);
+    height: 150px;
+    width: 300px;
+}
+
+
+
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/css/chapter16.css]: Avoid many empty lines, only one is enough

File: html-css/examples/css/chapter16.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    float: right;
+    margin-left: 10px;
+}
+
+img.aligning-medium {
+    width: 250px;
+    height: 250px;
+}
+
+
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/css/chapter16.css]: Remove redundant empty line, one line empty between css selectors group

File: html-css/examples/css/chapter15.css [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+    width: 960px;
+    margin: 0 auto;
+}
+
+.fixed>#content {
+    overflow: auto;
+    height: 100%;
+}
+
+.fixed> #header>#nav, .fixed>#content#feature, .fixed>#footer {
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/css/chapter15.css]: should be `.fixed > #header > #nav`, please read css style guide

Fix other places as well.

File: html-css/examples/chapter10.html (hunk 1 above)
Comment (lanhuong4867) [reply html-css/examples/chapter10.html]: Thank you for your comments. I will learn from experience

File: html-css/examples/css/chapter16.css (hunk 2 above)
Comment (lanhuong4867) [reply html-css/examples/css/chapter16.css]: Thank you for your comments. I will learn from experience

File: html-css/examples/css/chapter16.css (hunk 3 above)
Comment (lanhuong4867) [reply html-css/examples/css/chapter16.css]: Thank you for your comments. I will learn from experience

File: html-css/examples/css/chapter15.css (hunk 4 above)
Comment (lanhuong4867) [reply html-css/examples/css/chapter15.css]: Thank you for your comments. I will learn from experience

File: html-css/examples/chapter13-example.html [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+      width: 600px;
+      margin: 20px auto 20px auto;
+      font-weight: normal;
+    }
+
+    a {
+      color: #000000;
+      text-transform: uppercase;
+      text-decoration: none;
+      padding: 6px 18px 5px 18px;}
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/chapter13-example.html]: Wrong code format.

File: html-css/examples/chapter13-example.html [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+    <ul class="navigation">
+      <li><a href="#" class="on">Home</a></li>
+      <li><a href="#">For Sale</a></li>
+      <li><a href="#">Repairs</a></li>
+      <li><a href="#">About</a></li>
+      <li><a href="#">Contact</a></li>
+    </ul>
+
+    <p>
+      <img src="https://ghechua.net/wp-content/uploads/2022/05/meme-meo-doi-di-ra-di-2.jpg" alt="Fender Rhodes, Hohner Clavinet, and Wurlitzer EP200" width="100" height="100"/>
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/chapter13-example.html:79]: enter attributes to multiple lines

File: html-css/examples/chapter13.html [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+  <p class="limiting-p">class="limiting-h2"The Fender Stratocaster or "Strat" is one of the most popular electric guitars of all time ...</p>
+  <h2 class="limiting-h2">Gibson Les Paul</h2>
+  <p class="limiting-p">The Gibson Les Paul is a solid body electric guitar that was first sold in 1952 ...</p>
+  <br>
+
+  <!-- overflowing content -->
+  <h2>OVERFLOWING CONTENT</h2>
+  <h2>Fender Stratocaster</h2>
+  <p class="one">The Fender Stratocaster or "Strat" is one of the most popular electric guitars of all time, and its design has been copied by many guitar makers. It was designed by Leo... </p>
+  <h2> Gibson Les Paul</h2>
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/chapter13.html:45]: Redundant 1 space

File: html-css/examples/chapter14-example.html [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+      <p>
+        <label for="location" class="title">Your closest center:</label>
+        <select name="location" id="location">
+          <option value="ny">New York</option>
+          <option value="il">Chicago</option>
+          <option value="ca">San Francisco</option>
+        </select>
+      </p>
+      <span class="title">Are you a member?</span>
+      <label>     <input type="radio" name="member" value="yes" />     Yes</label>
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/chapter14-example.html]: Update code format

File: html-css/examples/chapter14.html [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+    <li> I have great faith in fools; self-confidence my friends call it.</li>
+    <li>All that we see or seem is but a dream within a dream.</li>
+    <li>I would define, in brief, the poetry of words as the rhythmical creation of Beauty.</li>
+  </ul>
+
+  <!-- table properties -->
+  <h2>TABLE PROPERTIES</h2>
+  <div class="table-body">
+    <h1>First Edition Auctions</h1>
+    <table  class="table-properties">
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/chapter14.html]: Redundant spaces

File: html-css/examples/chapter14.html [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+      <tr>
+        <th>Author</th>
+        <th>Title</th>
+        <th class="money">Reserve Price</th>
+        <th class="money"   >Current Bid</th>
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/chapter14.html]: Redundant spaces

File: html-css/examples/chapter15-example.html [hunk 11]
```diff
@@ -0,0 +1,191 @@
...
+          <h1><a href="">Fixed Gear Forever</a></h1>
+        </div>
+        <div class="intro grid_3 push_9">
+          <p class="date">16 APRIL 2011</p>
+          <p>The veloheld combines minimalist design with superb quality. Devoid of excessive graphics and gear shift components, the veloheld product range delights us with its beauty and simplicity ... </p>
+        </div>
+    </div>
+    
+    <div class="more-articles container_12">
+      <h2 class="grid_12"><a href="">More Articles</a></h2>
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/chapter15-example.html]: Update code format

File: html-css/examples/chapter15-example.html [hunk 12]
```diff
@@ -0,0 +1,191 @@
...
+      <div class="grid_3">
+        <img src="https://ghechua.net/wp-content/uploads/2022/05/meme-meo-doi-di-ra-di-2.jpg" alt="The road ahead" width="220" height="125" />
+        <p>     <a href="">On the Road: from the fixed gear fanatic's point of view</a>     </p>
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/chapter15-example.html]: Update code format

File: html-css/examples/chapter16-example.html [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+        background-image: url("images/icon.png");
+        padding-left: 20px;
+        background-repeat: no-repeat;
+      }
+    </style>
+</head>
+<body>
+  <div class="wrapper">
+    <div class="header">
+      <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a4/Magnolia_sieboldii_flower_1.jpg/1024px-Magnolia_sieboldii_flower_1.jpg" alt="Galerie Botanique" width="456" height="122" />
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/chapter16-example.html]: multiple lines for long attribute

File: html-css/examples/css/chapter13.css [hunk 14]
```diff
@@ -0,0 +1,254 @@
+/* box dimensions */
+.box {
+	height: 300px;
```
Comment (thanhnguyen-agilityio) [inline html-css/examples/css/chapter13.css:3]: 2 spaces for CSS code

**Comment by thanhnguyen-agilityio** [comment]
Hi @LanHuong142002, Please update this PR first, I need to make sure you do correct example + code style before continue code on practice.

**Pull request merged**